docs/appendices/release-notes/5.4.0.rst

```diff
@@ -6,6 +6,17 @@ Version 5.4.0
 
 Released on 2023-07-11.
 
+.. WARNING::
+
+    Version 5.4.0 introduced a regression which can cause some ``JOIN`` queries
```

Comment:
I'd tend towards a `Known Issues` section before `Breaking Changes`. I've the impression we're overusing the NOTE/WARNING labels a bit too much.

But up to you

Reply:
Yep, I agree, it's not a critical thing like other cases, thx!